MapifyEditor/Tools/TrackToolsWindow.cs

```diff
@@ -0,0 +1,1908 @@
+            {
+                CreateTrack(CurrentTrack.Curve[0].position, CurrentTrack.Curve[0].globalHandle2);
+            }
+
+            // Middle button.
+            GUI.enabled = CurrentTrack;
+
+            GUI.backgroundColor = EditorHelper.Cancel;
+            if (GUILayout.Button(new GUIContent("Destroy", "Destroys the current track"), widthOption))
+            {
+                DeleteTrack();
+            }
+
+            GUI.backgroundColor = Color.white;
+
+            // Forward button.
+            if (!CurrentTrack)
+            {
+                GUI.enabled = false;
+                tooltip = "No track";
+            }
+            else if (!IsAllowedCreation(CurrentTrack.GetGradeAtEnd()))
+            {
+                GUI.enabled = false;
+                tooltip = "Grade too steep for creation";
+            }
+            else if (_currentMode == CreationMode.Special && _currentSpecial == SpecialTrack.Connect2)
+            {
+                GUI.enabled = false;
+                tooltip = "Use the [New Track] button for this feature";
+            }
+            else
+            {
+                GUI.enabled = true;
+                tooltip = "Creates a track in front of the current one";
+            }
+
+            if (GUILayout.Button(new GUIContent(">>>", tooltip), smallWidth))
+            {
+                CreateTrack(CurrentTrack.Curve.Last().position, CurrentTrack.Curve.Last().globalHandle1);
+            }
+
+            GUI.enabled = true;
+
+            GUILayout.FlexibleSpace();
+            EditorGUILayout.EndHorizontal();
+
+            EditorGUILayout.BeginHorizontal();
+            GUILayout.FlexibleSpace();
+
+            GUI.backgroundColor = EditorHelper.Accept;
+            if (GUILayout.Button(new GUIContent("New track", "Creates a new track"), widthOption))
+            {
+                CreateNewTrack();
+            }
+
+            GUI.backgroundColor = Color.white;
+
+            GUILayout.FlexibleSpace();
+            EditorGUILayout.EndHorizontal();
+
+            EditorGUILayout.EndVertical();
+        }
+
+        private void CreationFoldoutContextMenu(Rect rect)
+        {
+            var menu = new GenericMenu();
+            menu.AddItem(new GUIContent("Reset current tab",
+                "Resets the current tab's values to default"),
+                false, () => ResetCreationSettings(false));
+            menu.AddItem(new GUIContent("Reset all tabs",
+                "Resets all tabs' values to default"),
+                false, () => ResetCreationSettings(true));
+            menu.DropDown(rect);
+        }
+
+        private void DrawEditingFoldout()
+        {
+            GUI.backgroundColor *= 1.1f;
+
+            _showEditing = EditorGUILayout.BeginFoldoutHeaderGroup(_showEditing,
+                new GUIContent("Editing", "Ways to edit tracks after they've been created"),
+                null, null);
```

Comment:
Null is the default style and menuAction, no need to specify them explicitly.
```suggestion
            _showEditing = EditorGUILayout.BeginFoldoutHeaderGroup(_showEditing,
                new GUIContent("Editing", "Ways to edit tracks after they've been created"));